guides/shipments/overview-of-shipments.md

```diff
@@ -0,0 +1,185 @@
+# Overview of shipments
+
+Solidus uses a flexible system to calculate shipping. It accommodates the full
+range of shipment pricing: from simple [flat
+rate](https://en.wikipedia.org/wiki/Flat_rate) calculations to more
+complex calculations related to product weight or type, the customer's shipping
+address, what shipping method is being used, and so on.
+
+If your store has complex shipping needs, you may find one of Solidus's existing
+shipping extensions, like [`solidus_active_shipping`][solidus-active-shipping]
+or [`solidus_shipstation`][solidus-shipstation], useful. Check out [the list of
+supported Solidus Extensions](https://extensions.solidus.io).
+
+This article provides a summary of shipping concepts. If you are interested in
+reading about example Solidus shipment setups see
+[Shipment setup examples](shipment-setup-examples.html.markdown).
```

Comment:
This link is broken

Reply:
Thanks for catching this @afdev82. Should be `shipment-setup-examples.md`. I will push an update soon.